sys/net/gnrc/network_layer/ipv6/nib/nib.c

```diff
@@ -1787,6 +1787,28 @@ static const char *_prio_string(uint8_t prio)
     return "invalid";
 }
 
+static gnrc_pktsnip_t *_build_ctxs(_nib_abr_entry_t *abr)
```

Comment:
Shouldn't there already be a function like this _somewhere_ in the code base?

Reply:
Only a function that adds [all options](https://github.com/RIOT-OS/RIOT/blob/master/sys/net/gnrc/network_layer/ipv6/nib/_nib-router.c#L225)

Reply:
To avoid code duplication, does it make sense to call the function at hand there as well?